web/tests/buddy_data.test.js

```diff
@@ -328,6 +328,54 @@ test("always show pm users", ({override_rewire}) => {
     assert.deepEqual(buddy_data.get_filtered_and_sorted_user_ids(""), [me.user_id, selma.user_id]);
 });
 
+test("show offline channel subscribers for small channels", ({override_rewire}) => {
+    // Add a bunch of users to hit max_size_before_shrinking, which will hide some
+    // offline users.
+    for (const i of _.range(2000, 2700)) {
+        const person = {
+            user_id: i,
+            full_name: `Human ${i}`,
+            email: `person${i}@example.com`,
+        };
+        people.add_active_user(person);
+    }
+    for (const user_id of _.range(2000, 2700)) {
+        set_presence(user_id, "offline");
+    }
```

Comment:
Overriding `max_size_before_shrinking` is probably a better plan here, too.

Maybe you can just rework these constants to be more easily mocked, like `message_fetch.consts`, which is designed to allow easily writing over a value in a test?

Reply:
Something like this:
```
+export const consts = {
+    max_channel_size_to_show_all_subscribers: 75,
+    max_realm_size_before_shrinking: 600,
+};
```

Reply:
Gonna leave this for later since it looks more of the existing test file could benefit from this.